src/game/__tests__/eop.test.js

```diff
@@ -41,27 +42,40 @@ describe('game', () => {
     );
   });
 
-  it('should start with the play phase', () => {
-    expect(players["0"].getState().ctx.phase).toBe('play');
+  it('should start in play stage', () => {
+    expect(players['0'].getState().ctx.activePlayers).toBeFalsy();
   });
 
+  it('should start with no activePlayers', () => {
```

Comment:
That's the same test as the one above. Maybe have only one test with a better description?